modules/security-context-constraints-requiring.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-requiring_{context}"]
+= Configuring a workload to require a specific SCC
+
+You can configure a workload to require a certain security context constraint (SCC). This is useful in scenarios where you want to pin to a specific SCC to the workload or if you want to prevent your required SCC from being preempted by another SCC in the cluster.
+
+To require a specific SCC, set the `openshift.io/required-scc` annotation on your workload. You can set this annotation on any resource that can set a pod manifest template, such as a deployment or daemon set.
+
+The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission will fail. An SCC is considered applicable to the workload if the user creating the pod or the pod's service account has `use` permissions for the SCC in the pod's namespace.
```

Comment:
```suggestion
The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission fails. An SCC is considered applicable to the workload if the user creating the pod or the pod's service account has `use` permissions for the SCC in the pod's namespace.
```

![Screenshot from 2023-08-28 15-05-12](https://github.com/openshift/openshift-docs/assets/57954076/1c1ed4e5-9a51-42e3-a25d-f20e8edf0547)

Reply:
s/must already exist/ must exist 

_must already exist_ seems redundant; _must exist_ is far more popular in the docs: 91 to 2, per Atom.